src/common.js

```diff
@@ -250,6 +258,13 @@ function wrap(cmd, fn, options) {
             return accum;
           }
         }, []);
+        // Convert ShellStrings to regular strings
+        args = args.map(function(arg) {
+          if (arg.constructor.name === 'String') {
```

Comment:
That's what I'm doing in the next line, isn't it? This prevents us from calling toString() on things like vanilla objects and integers, which wouldn't be ideal (chmod takes an integer, touch sometimes takes an object containing options like -r)